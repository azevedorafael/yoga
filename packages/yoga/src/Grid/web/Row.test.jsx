import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider } from '../..';
import { Row } from '.';

describe('<Row />', () => {
  describe('Snapshots', () => {
    it('should match snapshot with default Row', () => {
      const { container } = render(
        <ThemeProvider>
          <Row />
        </ThemeProvider>,
      );
      expect(container).toMatchSnapshot();
    });
  });
});
