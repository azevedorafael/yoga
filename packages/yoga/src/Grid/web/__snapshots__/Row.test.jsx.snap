// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> Snapshots should match snapshot with default Row 1`] = `
.c0 {
  display: grid;
  box-sizing: border-box;
  grid-template-columns: repeat(12,1fr);
}

@media (min-width:0px) {
  .c0 {
    grid-column-gap: 16px;
  }
}

@media (min-width:1024px) {
  .c0 {
    grid-column-gap: 24px;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
